leshan-integration-tests/logback-leshan-test.xml

```diff
@@ -28,7 +28,7 @@ Contributors:
 		</encoder>
 	</appender>
 
-	<root level="ERROR">
+	<root level="DEBUG">
```

Comment:
I guess it is not expected to be changed ?

Reply:
Yep. I'll revert it.